uint8 planner_type
uint8 CLASSIC=0
uint8 NEURAL=1
